schematools/cli.py

```diff
@@ -174,7 +195,10 @@ def import_ndjson(db_url, schema_url, dataset_id, table_name, ndjson_path):
     """Import an NDJSON file into a table."""
     # Add batching for rows.
     engine = _get_engine(db_url)
-    dataset_schema = schema_def_from_url(schema_url, dataset_id)
+    try:
+        dataset_schema = schema_def_from_url(schema_url, dataset_id)
+    except KeyError:
+        raise click.BadParameter(f"Schema {dataset_id} not found.")
```

Comment:
in mijn GeoJSON PR heb ik deze code dry gemaakt. Kunnen we later ook aanpassen :-)